app/uk/gov/hmrc/plasticpackagingtax/returns/models/request/JourneyAction.scala

```diff
@@ -61,7 +63,13 @@ class JourneyAction @Inject() (returnsConnector: TaxReturnsConnector, auditor: A
           }
           .getOrElse {
             auditor.newTaxReturnStarted()
-            returnsConnector.create(TaxReturn(id))
+            // TODO: get this from the PPT obligations API
```

Comment:
Shall we do this in this PR, or in the first page PR?

Reply:
I'm happy for you to include this in your PR or to get another ticket raised and do it as part of that one. The default values here should be good enough for the rest of the UI tickets in this flow.